ios/FluentUI/Controls/SearchBar.swift

```diff
@@ -23,6 +24,7 @@ public extension Colors {
             public static var cancelButton: UIColor = LightContent.text
             public static var clearIcon = UIColor(light: iconOnAccent, dark: textSecondary)
             public static var placeholderText = UIColor(light: textOnAccent, dark: textSecondary)
+            public static var progressSpinner: UIColor = placeholderText
```

Comment:
seems incorrect to me. we should show primaryTint10 color for light mode if background is app primary color (like communicationblue)

Reply:
updated the color. But i could not identity the difference in color for light mode on primary background in demo app.

---
In reply to: [465968168](https://github.com/microsoft/fluentui-apple/pull/174#discussion_r465968168) [](ancestors = 465968168)